_sass/_type.scss

```diff
@@ -110,3 +110,31 @@ h4 {
     color: $dark-blue;
   }
 }
+
+.awards-list {
+  list-style-position: outside;
+  
+  li {
+    margin-bottom: 0.5rem;
+    line-height: 1.3;
+
+    &::marker {
+      color: rgba(251,147,6,0.75);
```

Comment:
Should use the Sass functions to lighten the `$orange` color variable: `color: lighten($orange, 25%);`

Reply:
Fixed.